string first_name
string last_name
# from 1-99
int8 age
float32 marks